packages/samples/test/output/azure/resource-manager/arm-scenarios/tracked/@azure-tools/typespec-autorest/2021-10-01-preview/openapi.json

```diff
@@ -241,6 +241,14 @@
                 "type": "integer",
                 "format": "int32",
                 "description": "The Retry-After header can indicate how long the client should wait before polling the operation status."
+              },
+              "Azure-AsyncOperation": {
```

Comment:
Looks like by default we output both header. Should the samples be updated to use a single recommended pattern one or the other?

Reply:
We have options for all 3 patterns (Azure-AsyncOperations only,  Location only, and combined).  The template default uses the recommended async implementation for RPaaS, and matches the existing `final-state-via`  value.